source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/digit/rough_env_cfg.py

```diff
@@ -0,0 +1,353 @@
+        weight=-0.1,
+        params={"asset_cfg": SceneEntityCfg("robot", joint_names=[".*_toe_a", ".*_toe_b"])},
+    )
+    joint_deviation_arms = RewardTermCfg(
+        func=mdp.joint_deviation_l1,
+        weight=-0.2,
+        params={
+            "asset_cfg": SceneEntityCfg("robot", joint_names=".*_arm_.*"),
+        },
+    )
+
+    undesired_contacts = RewardTermCfg(
+        func=mdp.undesired_contacts,
+        weight=-0.1,
+        params={
+            "sensor_cfg": SceneEntityCfg("contact_forces", body_names=[".*_rod", ".*_tarsus"]),
+            "threshold": 1.0,
+        },
+    )
+
+
+@configclass
+class DigitObservations:
+    @configclass
+    class PolicyCfg(ObservationGroupCfg):
+        base_lin_vel = ObservationTermCfg(
+            func=mdp.base_lin_vel,
+            noise=Unoise(n_min=-0.1, n_max=0.1),
+        )
+        base_ang_vel = ObservationTermCfg(
+            func=mdp.base_ang_vel,
+            noise=Unoise(n_min=-0.2, n_max=0.2),
+        )
+        projected_gravity = ObservationTermCfg(
+            func=mdp.projected_gravity,
+            noise=Unoise(n_min=-0.05, n_max=0.05),
+        )
+        velocity_commands = ObservationTermCfg(
+            func=mdp.generated_commands,
+            params={"command_name": "base_velocity"},
+        )
+        joint_pos = ObservationTermCfg(
+            func=mdp.joint_pos_rel,
+            params={"asset_cfg": SceneEntityCfg("robot", joint_names=LEG_JOINT_NAMES + ARM_JOINT_NAMES)},
+            noise=Unoise(n_min=-0.01, n_max=0.01),
+        )
+        joint_vel = ObservationTermCfg(
+            func=mdp.joint_vel_rel,
+            params={"asset_cfg": SceneEntityCfg("robot", joint_names=LEG_JOINT_NAMES + ARM_JOINT_NAMES)},
+            noise=Unoise(n_min=-1.5, n_max=1.5),
+        )
+        actions = ObservationTermCfg(func=mdp.last_action)
+        height_scan = ObservationTermCfg(
+            func=mdp.height_scan,
+            params={"sensor_cfg": SceneEntityCfg("height_scanner")},
+            noise=Unoise(n_min=-0.1, n_max=0.1),
+            clip=(-1.0, 1.0),
+        )
+
+        def __post_init__(self):
+            self.enable_corruption = True
+            self.concatenate_terms = True
+
+    # Observation groups:
+    policy: PolicyCfg = PolicyCfg()
+
+
+@configclass
+class TerminationsCfg:
+    """Termination terms for the MDP."""
+
+    time_out = TerminationTermCfg(func=mdp.time_out, time_out=True)
+    base_contact = TerminationTermCfg(
+        func=mdp.illegal_contact,
+        params={
+            "sensor_cfg": SceneEntityCfg("contact_forces", body_names="torso_base"),
+            "threshold": 1.0,
+        },
+    )
+
+
+@configclass
+class ActionsCfg:
+    """Action specifications for the MDP."""
+
+    joint_pos = mdp.JointPositionActionCfg(
+        asset_name="robot",
+        joint_names=LEG_JOINT_NAMES + ARM_JOINT_NAMES,
+        scale=0.5,
+        use_default_offset=True,
+    )
+
+
+@configclass
+class EventCfg:
```

Comment:
The events seem a duplication of `LocomotionVelocityEnvCfg` with just the renaming of base link. If that's the only/main change, I suggest not duplicating and just renaming in the inherited config.